src/send/send_caliper_events.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import fileinput
+import json
+import time
+
+
+def api_handler(data):
+    AUTHORIZATION = 'Authorization'
+    BASIC = 'Basic '
+    MIME_TYPE_JSON = 'application/json'
+    CONTENT_TYPE = 'Content-type'
+    headers = {CONTENT_TYPE: MIME_TYPE_JSON, AUTHORIZATION: BASIC + TOKEN}
+    response = None
+    try:
+        response = requests.post(url, data=data, headers=headers)
```

Comment:
FWIW, the "[Advanced Usage](http://docs.python-requests.org/en/master/user/advanced/)" page from the requests documentation talks about reusing connections (via a connection pool).